server/src/test/scala/sbt/server/ProtocolTest.scala

```diff
@@ -621,13 +624,13 @@ class ProtocolTest {
           case ObjectToJson =>
             val json = addWhatWeWerePickling(t)(SerializedValue(t)(format.pickler))
             // IO.write(path, json.toString, IO.utf8)
-            if (path.exists) sys.error(s"$path exists already")
+            if (path.exists) () // Just ignore things we write twice, reading into different types should work.
```

Comment:
I originally intended oneWayTrip to write once per version and to ensure that JSON is deserializable.
It doesn't have to be identical if we used `Option[A]` to grow the case class.